### DISCUSSION TITLE
VSCode plugin compaibility

### DISCUSSION THREAD

**Discussion by MeStrak** [LefterisXris/CodeTour#67] [Q&A] [answered]
Hi there, I noticed that you renamed this plugin CodeTour to align with the existing VSCode plugin.

My team use a mixture of VSCode and IntelliJ and I would like to start using these plugins. Have you already established compatibility? I mean if we write a tour in VSCode will it work in both?

Thanks!

**Comment by LefterisXris** [comment, marked as answer]
Hello @MeStrak! 

Thanks for reaching out. Indeed the compatibility between those plugins is one of the most important features and I tried to support it.
Tours generated by VS Code's plugin, seem to be compatible with IntelliJ CodeTour plugin, but to be honest I haven't tried the vice-versa.
However I can assure you, that in case there is any compatibility issue, I will deal with it asap.
To conclude, I suggest you try it with your team and let me know if you need further support for any issue or question that may arise.

Cheers!

**Reply by MeStrak** [reply]
Thanks!
